src/qibolab/dummy.py

```diff
@@ -60,6 +62,11 @@ def create_dummy(with_couplers: bool = True):
     channels["twpa"].local_oscillator = twpa_pump
 
     qubits, couplers, pairs = load_qubits(runcard)
+    if with_kernels:
+        kernels = Kernels.load(path=extras_folder / "kernels.npz")
+        for q in kernels:
+            qubits[q].kernel = kernels[q]
+
```

Comment:
Since this is already repeated (and different) between here and https://github.com/qiboteam/qibolab_platforms_qrc/pull/106, why not including it in `load_qubits`?

You could add a `kernels_path=None` argument to `load_qubits`. If `None`, you won't load kernels, otherwise you will, using that path (that is the only external input for this block).

(more or less as it was before, with the only difference in `create()`, since you wouldn't load the path from the runcard any longer)